po/meson.build

```diff
@@ -0,0 +1,23 @@
+i18n = import('i18n')
+
+i18n.gettext(
+  gettext_package,
+  args: [
+    # Extract translations from generated file.
+    # Unfortunately, there is currently no way to express the dependency
+    # so one will need to build the project before updating the po template.
+    # https://github.com/mesonbuild/meson/issues/1733
+    brush_settings_headers[1].full_path(),
```

Comment:
Yuck. Maybe we need to just replicate the Meson’s `i18n` module with `run_target`.

Reply:
Is it possible to extract the translations from the source code which is used to generate this file?

Reply:
I believe gettext currently doesn't support extracting messages from [JSON files](https://github.com/mypaint/libmypaint/blob/master/brushsettings.json). I only found one comment in the issue tracker mentioning it: https://savannah.gnu.org/bugs/?48903#comment1

I think we would need to either switch to XML, since ITS works there, or [go back](https://github.com/mypaint/libmypaint/commit/e29c78b9cbd71b4668f87272158b296abfcc7d68) to listing the brushes in the Python script that generates the C header.

Reply:
Turns out autofoo does not actually handle the locations transparently. That’s what `po/update_translations.py` does. So I just recreated the logic directly inside Meson script. I had to do another hack because of https://github.com/mesonbuild/meson/issues/2320 but I am reasonably happy with it now.